metrics/cli/test/flutter/adapter/flutter_cli_service_adapter_test.dart

```diff
@@ -12,7 +12,8 @@ void main() {
   group("FlutterCliServiceAdapter", () {
     const path = './test';
     final flutterCli = _FlutterCliMock();
-    final flutterCliServiceAdapter = FlutterCliServiceAdapter(flutterCli);
+    final flutterAdapter = FlutterCliServiceAdapter(flutterCli);
```

Comment:
```suggestion
    final flutterService = FlutterCliServiceAdapter(flutterCli);
```
Would it be a bit more clear?